packages/styles/scss/patterns/sub-patterns/content-section/_content-section.scss

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../../globals/imports';
+
+@mixin themed-items {
+  color: $text-01;
+  background: $ui-background;
+}
+
+@mixin content-section {
+  .#{$prefix}--content-section {
+    @include carbon--make-row;
+
+    &__heading {
```

Comment:
Now there is an interesting relationship between spacing that we need to take into account between the `heading`, and the pattern as a whole.

`@lg` breakpoint the pattern as a whole has only `48px` spacing top. While the heading has as `64px` margin-top. Because of the `48px` that's actually more like `16px` top that needs to be added here. I believe that would be a `layout-01`.

![image](https://user-images.githubusercontent.com/3793636/74487275-aa6ad880-4e84-11ea-9f13-854a6fd0e753.png)

With that said that changes things for the smaller breakpoints as well. We could update this on a different break point, or we could just take it into account when adding spacing to the pattern top as a whole. Something to think about.

`@sm --> layout-01`
`@lg --> layout-03`